go/oasis-net-runner/fixtures/fixtures.go

```diff
@@ -0,0 +1,79 @@
+// Package fixtures provides network configuration fixtures.
+package fixtures
+
+import (
+	"encoding/json"
+	"math"
+
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/oasislabs/oasis-core/go/common"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/oasis"
+)
+
+const (
+	cfgFile = "fixture.file"
+
+	cfgEpochtimeMock       = "fixture.default.epochtime_mock"
```

Comment:
Since these flags are only used in `default.go` I would rather have them defined there?